engine/apps/slack/models/slack_usergroup.py

```diff
@@ -104,9 +104,8 @@ def update_members(self, slack_ids):
         self.save(update_fields=("members",))
 
     def get_users_from_members_for_organization(self, organization):
-        return organization.users.filter(
-            slack_user_identity__slack_id__in=self.members, role__in=[Role.ADMIN, Role.EDITOR]
-        )
+        users = organization.users.filter(slack_user_identity__slack_id__in=self.members)
```

Comment:
Probably more efficient to use the regex lookup here to get only users having the `CHATOPS_WRITE` permission?

Reply:
Good call. Refactored this in [0d0dad1](https://github.com/grafana/oncall/pull/634/commits/0d0dad1a66b3183f947a72be548bac7d19610462)